src/Archive/Archives.ts

```diff
@@ -10,9 +10,9 @@ export interface Spec {
 
 export enum SpecType {
   NEXMOCK = 'NexMock',
-  HAR = 'HAR',
-  OPENAPI = 'OpenAPI',
-  POSTMAN = 'Postman'
+  HAR = 'har',
+  OPENAPI = 'openapi',
+  POSTMAN = 'postman'
```

Comment:
It is a dirty hack that has nothing to do with the actual issue. First of all, `type` is not used to upload files anymore. Secondary, `type` argument is cast to `SpecType` explicitly, while it is not possible without a transformation (see `Helpers.selectEnumValue()`).

Reply:
> First of all, type is not used to upload files anymore.

What do you mean? We should not support this param anymore?

Reply:
There is no place in the `upload` method where it is used except the following runtime check: `this.ALLOWED_SPECS.includes(spec.type)`. So, why don't check type on compile-time?

Reply:
@BMNTeam no reaction for 4 days, please make sure that your team member is aware about https://team-1602965683919.atlassian.net/wiki/spaces/TEST/pages/410058861/Pull+request+live+cycle